libs/core/kiln_ai/datamodel/task.py

```
@@ -134,6 +133,25 @@ def run_config(self) -> RunConfig:
             run_config_properties=self.run_config_properties,
         )
 
+    # Previously we didn't store structured_output_mode in the run_config_properties. Updgrade old models when loading from file.
+    @model_validator(mode="before")
+    def upgrade_old_entried(cls, data: dict, info: ValidationInfo) -> dict:
```

Pylint (3.3.7)

[error] Method 'upgrade_old_entried' should have "self" as first argument

(E0213)